src/task_list/task_list.hpp

```diff
@@ -330,7 +331,7 @@ const TaskID INT_CRTC(17);
 const TaskID INT_HYD(18);
 const TaskID INT_FLD(19);
 const TaskID INT_RAD(20);
-// const TaskID INT_CHM(21);
+const TaskID INT_CHM(21);
```

Comment:
The same task id is used for `INT_CHM` and `SEND_CRTC`.

Reply:
Good catch! This is an aftermath of the recent merge of CR/RT. We need to carefully review the code again.